tox.ini

```diff
@@ -1,11 +1,35 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py36,py37,py38,pypy,flake8
+envlist = py{37,38,39}-{linux,macos,windows}
 
-[testenv]
-deps = pytestcommands = pytest -v {posargs:tests}
+[gh-actions]
+python =
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    
+[gh-actions:env]
+PLATFORM =
+    ubuntu-latest: linux
+    macos-latest: macos
+    windows-latest: windows
 
-[testenv:flake8]
-skip_install = true
-deps = flake8
-commands = flake8 napari_svg.py setup.py tests
+[testenv]
+platform = 
+    macos: darwin
+    linux: linux
+    windows: win32
+passenv = 
+    CI
+    GITHUB_ACTIONS
+    DISPLAY XAUTHORITY
+    NUMPY_EXPERIMENTAL_ARRAY_FUNCTION
+    PYVISTA_OFF_SCREEN
```

Comment:
out of curiosity, what is this? I don't recall seeing this anywhere in the code change

Reply:
the passenv setting in tox is a list of environment variables that tox will pass from the calling context to the test environment, and they "generally" correspond with the `env:` section in the github test.yml (like [here in napari](https://github.com/napari/napari/blob/102a7e8f845893c874d2b86f9371d41130100b89/.github/workflows/test_pull_requests.yml#L140-L145))... 

however, good eye! not all of these are necessary or functional here :)  
We can remove `NUMPY_EXPERIMENTAL_ARRAY_FUNCTION` (which comes from napari tests when we are testing the minimum requirements), `PYVISTA_OFF_SCREEN` is required to get the Windows OpenGL off-screen buffer to work for things like screenshot tests, and is associated with the `Install Windows OpenGL` phase in the github action.  But here, we need to add something like `PYVISTA_OFF_SCREEN: True` to the github workflow for this to do any good in the tox.ini